### PR TITLE
Drop support for Python 3.8 and 3.9

### PR DESCRIPTION
**Pull request by CyclingNinja**
Removes old python versions from builds and min required python

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -11,6 +11,8 @@ classifiers =
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
```
Comment (astrofrog) [inline setup.cfg:12]: Could you remove 3.8 and 3.9 here?

**Review by astrofrog — CHANGES_REQUESTED** [review]
Small comment below - also could you check whether the documentation needs updating in the installation instructions? (regarding the minimum version of Python)

**Review by astrofrog — APPROVED** [review]
Thanks! I reworded the title just as this gets used for the user-facing change notes

File: setup.cfg (hunk 1 above)
Comment (dhomeier) [reply setup.cfg:12]: These have been forgotten

**Comment by dhomeier** [comment]
A bit puzzled why `py310-test-dev` is still running here, but not for `glue-qt` – is `astropy>=0.0.dev0` resolved differently from `git+https://github.com/astropy/astropy` (weekly wheels or something)?
Either way this may still start to fail here as well once the former are updated to astropy HEAD.
Also does the LTS test need no replacement?

**Comment by dhomeier** [comment]
> Either way this may still start to fail here as well once the former are updated to astropy HEAD.

@astrofrog, @CyclingNinja, `astropy>=0.0.dev0` is now getting `astropy==6.1.3` installed on `py310-test-dev`, so one dev test should probably be pushed to py312.

**Pull request merged**